lib/ansible/modules/windows/win_domain_controller.py

```diff
@@ -33,24 +22,40 @@
 options:
   dns_domain_name:
     description:
-      - when C(state) is C(domain_controller), the DNS name of the domain for which the targeted Windows host should be a DC
+      - When C(state) is C(domain_controller), the DNS name of the domain for which the targeted Windows host should be a DC.
   domain_admin_user:
     description:
-      - username of a domain admin for the target domain (necessary to promote or demote a domain controller)
+      - Username of a domain admin for the target domain (necessary to promote or demote a domain controller).
     required: true
   domain_admin_password:
     description:
-      - password for the specified C(domain_admin_user)
+      - Password for the specified C(domain_admin_user).
     required: true
   safe_mode_password:
     description:
-      - safe mode password for the domain controller (required when C(state) is C(domain_controller))
+      - Safe mode password for the domain controller (required when C(state) is C(domain_controller)).
   local_admin_password:
     description:
-      - password to be assigned to the local C(Administrator) user (required when C(state) is C(member_server))
+      - Password to be assigned to the local C(Administrator) user (required when C(state) is C(member_server)).
+  read_only:
+    description:
+      - Whether to install the domain controller as a read only replica for an
+        existing domain.
+    type: bool
+    choices:
```

Comment:
`choices:` no longer required

Reply:
I thought so too but when I generated the docs the options never came up. Might be a bug though so I will investigate

Reply:
I looked into it, was a bug and should be fixed with https://github.com/ansible/ansible/pull/34074.